middleware/errors.js

```diff
@@ -54,10 +50,22 @@ const getContent = (err, translate) => {
     content.timeToWait = rateLimitsConfig.rateLimits.submissions.windowSizeInMinutes;
     content.postTimeToWait = (translate && translate('errors.submission-rate-limit.post-time-to-wait'));
   }
-
-  err.code = err.code || 'UNKNOWN';
-  err.status = err.status || 500;
-
+  if (err.code === 'UNKNOWN' || err.code === 'INTERNAL_SERVER_ERROR') {
+    err.status = 500;
+    err.template = '500';
+    err.title = (translate && translate('errors.500.title'));
+    err.message = (translate && translate('errors.500.description'));
+    err.header = (translate && translate('errors.500.header'));
+    err.paragraph1 = (translate && translate('errors.500.paragraph1'));
+    err.paragraph2 = (translate && translate('errors.500.paragraph2'));
+    err.paragraph3 = (translate && translate('errors.500.paragraph3'));
+    content.title = (translate && translate('errors.500.title'));
+    content.message = (translate && translate('errors.500.description'));
+    content.header = (translate && translate('errors.500.header'));
+    content.paragraph1 = (translate && translate('errors.500.paragraph1'));
+    content.paragraph2 = (translate && translate('errors.500.paragraph2'));
+    content.paragraph3 = (translate && translate('errors.500.paragraph3'));
+  }
```

Comment:
not your fault but why does this all have to be set manually for each translation? 
can you just use a markdown file?

Reply:
Done